contrib/tests/actions/chains/test_packs_pack.yaml

```diff
@@ -120,7 +120,7 @@ chain:
         params:
             env:
               ST2_AUTH_TOKEN: "{{token}}"
-            cmd: "st2 run packs.load register=all | grep -c 'Registered' | grep 3"
+            cmd: "st2 run packs.load register=all | grep -c 'Registered' | grep 4"
```

Comment:
@lakshmi-kannan: yes, a few things currently are hard-coded and can be improved. But I wanted first of all to fix current flow, to get it back to the build process, as this is important. So I tried to do minimum changes.